src/main/app/src/app/zaken/zaken.service.ts

```diff
@@ -179,6 +179,7 @@ export class ZakenService {
 
   verdelenVanuitLijst(
     uuids: string[],
+    screenEventResourceId: string,
```

Comment:
Why is the `screenEventResourceId` already sent now on TEST but not locally? Is this code only used locally? I'm a bit confused.. ;-)

Reply:
It's also not sent on Test :-)

Reply:
Ah sorry, you are right. I was looking at taken verdelen again.. 

I see your point. If we don't use it on the frontend it is weird to keep it in there. Then again, at the moment our integration tests rely on this feature so maybe for another day..